lib/pure/httpclient.nim

```diff
@@ -150,6 +150,33 @@ proc code*(response: Response | AsyncResponse): HttpCode
   ## corresponding ``HttpCode``.
   return response.status[0 .. 2].parseInt.HttpCode
 
+proc contentType*(response: Response | AsyncResponse): string =
+  ## Retrieves the specified response's ``Content type``.
```

Comment:
Nitpick: don't use \`\` in documentation where quotes would be more appropriate.

Reply:
```
## Retrieves the specified response's content type. This is effectively the value of the "content-type" header.
```

You don't even need to emphasise the "content type" really.